src/components/va-accordion/va-accordion-item.css

```diff
@@ -0,0 +1,45 @@
+@import '../../mixins/focusable.css';
+
+:host {
```

Comment:
From [Stencil's docs on Styling & IE Support](https://stenciljs.com/docs/styling#ie-support):

> - Global CSS variables can only be declared in `:root` or `html`, they can't be dynamic.
> - CSS variables within a component can ONLY be defined within a `:host(...)` selector.
> - CSS variables within a component can be consumed (`var(--thing)`) in any selector.

So, this will work in IE.